style.css

```diff
@@ -81,5 +86,216 @@ a:active {
   justify-content: center;
   justify-content: space-around;
   list-style: none;
-  padding-top: 10%;
+  padding-left: 102.42px;
+  padding-right: 102.42px;
+}
+
+.down-btn {
+  display: flex;
+  justify-content: center;
+  list-style: none;
+  margin-bottom: 0%;
+  padding-left: 0%;
+}
+
+.recent-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.recent-work-title {
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  margin-bottom: 0%;
+}
+
+#section-2-green-title-line {
+  margin-top: 5%;
+}
+
+.work-1-container {
+  width: 327px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  background: #ebf0ee;
+  border-radius: 8px;
+  margin-top: 62px;
+}
+
+.project-image {
+  width: 100%;
+  height: 252px;
+}
+
+.work-bottom-cont {
+  border: 1px solid #fff;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 8px;
+}
+
+.project-title {
+  text-align: center;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  margin-bottom: 0;
+}
+
+.Language-tags {
+  list-style: none;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding-left: 0;
+  margin-top: 0%;
+}
+
+.lang-tag {
+  background-color: #ebf0ee;
+  padding: 8px;
+  border-radius: 4px;
+  margin: 8px;
+}
+
+.project-button {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  text-align: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  border: 0;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.about-heading {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  order: 0;
+  margin-bottom: 24px;
+}
+
+.about-me-para {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  margin: 0 12px 0 12px;
+  order: 1;
+}
+
+.resume-butto-cont {
+  order: 2;
+  margin-top: 24px;
+}
+
+.resume-button {
```

Comment:
- [ ] I think it's better to implement the button interactions (enable, hover, etc.). It's a project requirement.